napari_plugin_manager/qt_plugin_dialog.py

```diff
@@ -1178,13 +1187,35 @@ def _setup_ui(self):
         visibility_direct_entry = not running_as_constructor_app()
         self.direct_entry_edit = QLineEdit(self)
         self.direct_entry_edit.installEventFilter(self)
-        self.direct_entry_edit.setPlaceholderText(
-            trans._('install by name/url, or drop file...')
-        )
+        self.direct_entry_edit.returnPressed.connect(self._install_packages)
         self.direct_entry_edit.setVisible(visibility_direct_entry)
-        self.direct_entry_btn = QPushButton(trans._("Install"), self)
+        self.direct_entry_btn = QToolButton(self)
         self.direct_entry_btn.setVisible(visibility_direct_entry)
         self.direct_entry_btn.clicked.connect(self._install_packages)
+        self.direct_entry_btn.setText(trans._("Install"))
+        self.direct_entry_btn.setToolButtonStyle(Qt.ToolButtonTextBesideIcon)
+        self.direct_entry_btn.setPopupMode(QToolButton.MenuButtonPopup)
```

Comment:
Checking this again, I think it makes sense to move this line to be under the `IS_NAPARI_CONDA_INSTALLED` validation (around line 1216). In that way, the highlighted arrow will only be shown when there are options to select. Currently the arrow ends up showing for both the conda and pip cases.

Reply:
You are correct!